jobs/integration/validation.py

```diff
@@ -454,12 +453,8 @@ def validate_status_messages(model):
     action = await workers.units[0].run('lspci -nnk')
     nvidia = True if action.results['Stdout'].lower().count("nvidia") > 0 else False
 
-    # See what the runtime is set to
-    config = await workers.get_config()
-    runtime = config['docker_runtime']['value']
-
     master_unit = model.applications['kubernetes-master'].units[0]
-    if not nvidia or runtime == 'apt' or runtime == 'upstream':
+    if not nvidia:
```

Comment:
Before, on nvidia-enabled hardware, it was possible to opt out of nvidia support by setting docker_runtime=apt. Now it looks like there's no way to opt out. Will that be a problem?

LXD comes to mind. I've seen a lot of cases where LXD deployments detect nvidia hardware on the host, but are unable to use it, and that causes problems. The solution has been to set docker_runtime=apt. Do we have a new solution?

Reply:
@Cynerva This can still be done with the Docker subordinate (all the previous option remain, it's just about making these CDK tests generic).  If we think a `gpu_disable`-esque flag would be useful in the containerd subordinate, that's no issue.

Reply:
> If we think a gpu_disable-esque flag would be useful in the containerd subordinate, that's no issue.

I guess I'd test containerd on LXD to see if it's needed. That's the only use case I'm aware of, at least.

Reply:
+1 I'll get that added to the charm config.